### ISSUE TITLE
NtOS error when attempting to create a culture bottle from the PanD.E.M.I.C. 2200 on Delta Station

### ISSUE THREAD

**Issue by HaddamSussein** [tgstation/tgstation#70896]
## Round ID:
[193348](https://scrubby.melonmesa.com/round/193348), the map is Delta Station

## Testmerges:
[#70409: 'Chefs can now kiss food they make to make it with love. (And a lot of cooking refactoring)' by MrMelbert at commit 5a45fb6c4a](https://www.github.com/tgstation/tgstation/pull/70409)
[#70874: 'Fix halloweens races' by MrMelbert at commit e1c60d152c](https://www.github.com/tgstation/tgstation/pull/70874)

## Reproduction:
I joined the round as a Virologist, filled a beaker with 15 units of blood and put it in the PanD.E.M.I.C. machine. It doesn't have any diseases or symptoms in it. When I click the **Create culture bottle** button, the PanD.E.M.I.C. window immediately transitions to a NtOS error screen and I'm presented with the following error message:

TypeError: Unable to get property 'index' of undefined or null reference
   at onClick (https://tgstation13.download/byond/assets/5e/asset.5e453a3ff0cfe8d8b4fb1985848c0be1.js:1:1283107)
   at L.onClick (https://tgstation13.download/byond/assets/5e/asset.5e453a3ff0cfe8d8b4fb1985848c0be1.js:1:277032)
   at q (https://tgstation13.download/byond/assets/5e/asset.5e453a3ff0cfe8d8b4fb1985848c0be1.js:1:173630)
   at Anonymous function (https://tgstation13.download/byond/assets/5e/asset.5e453a3ff0cfe8d8b4fb1985848c0be1.js:1:173225)
User Agent: Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; Zoom 3.6.0)
State: {"ckey":"haddamsussein","interface":"Pandemic","window":{"key":"[0x2009a63]-main","size":null,"fancy":1,"locked":1}}

**Comment by Mooshimi** [comment]
Yeah reproduction is: Any blood without a virus/symptoms present -> create culture button.

```
TypeError: Unable to get property 'index' of undefined or null reference
   at onClick (https://tgstation13.download/byond/assets/2e/asset.2e7e48b204c2802228eb90e3542c39ec.js:1:1287312)
   at L.onClick (https://tgstation13.download/byond/assets/2e/asset.2e7e48b204c2802228eb90e3542c39ec.js:1:277049)
   at q (https://tgstation13.download/byond/assets/2e/asset.2e7e48b204c2802228eb90e3542c39ec.js:1:173630)
   at Anonymous function (https://tgstation13.download/byond/assets/2e/asset.2e7e48b204c2802228eb90e3542c39ec.js:1:173225)
User Agent: Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; Tablet PC 2.0; Zoom 3.6.0; Zoom 3.6.0)
State: {"ckey":"mooshimi","interface":"Pandemic","window":{"key":"[0x20089c8]-main","size":null,"fancy":1,"locked":0}}
```

**Comment by timothymtorres** [comment]
Relevant code:

https://github.com/tgstation/tgstation/blob/4d6a8bc5371eb61f9adb7c4dd1b8c441d3ae9251/code/modules/reagents/chemistry/machinery/pandemic.dm#L155-L212

**Comment by SyncIt21** [comment]
@Mooshimi this was fixed by #73613. The create culture button is greyed out now if no virus culture is present in the bottle.
